ansible/all-in-one/roles/xsnippet/templates/xsnippet.yml.j2

```diff
@@ -0,0 +1,101 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: xsnippet
+spec:
+  containers:
+    - name: db
+      image: "{{ xsnippet_db_image }}"
+      env:
+        - name: POSTGRES_DB
+          value: xsnippet
+        - name: POSTGRES_USER
+          value: xsnippet
+        - name: POSTGRES_PASSWORD
+          value: xsnippet
+      volumeMounts:
+        - mountPath: /var/lib/postgresql/data
+          name: database-state
+    - name: api
+      image: "{{ xsnippet_api_image }}"
+      env:
+        - name: ROCKET_HOST
+          value: "{{ xsnippet_host|default('0.0.0.0') }}"
+        - name: ROCKET_PORT
+          value: "{{ xsnippet_port|default('8000') }}"
+        - name: ROCKET_DATABASE_URL
+          value: "postgresql://xsnippet:xsnippet@127.0.0.1:5432/xsnippet"
+        - name: ROCKET_SYNTAXES
+          value: '{{ xsnippet_syntaxes|to_json }}'
+    - name: web
+      image: "{{ xsnippet_web_image }}"
+      volumeMounts:
+        - mountPath: /www/data
+          name: spa-assets
+          readOnly: true
+        - mountPath: /etc/nginx/conf.d
+          name: nginx-conf.d
+          readOnly: true
+        - mountPath: /etc/nginx/nginx.conf
+          name: nginx-conf
+          readOnly: true
+{% if xsnippet_api_https %}
+        - mountPath: /etc/nginx/xsnippet_api_ssl.cert
+          name: api-ssl-cert
+          readOnly: true
+        - mountPath: /etc/nginx/xsnippet_api_ssl.key
+          name: api-ssl-key
+          readOnly: true
+{% endif %}
+{% if xsnippet_spa_https %}
+        - mountPath: /etc/nginx/xsnippet_spa_ssl.cert
+          name: spa-ssl-cert
+          readOnly: true
+        - mountPath: /etc/nginx/xsnippet_spa_ssl.key
+          name: spa-ssl-key
+          readOnly: true
+{% endif %}
+      ports:
+        - containerPort: 80
+          hostPort: {{ xsnippet_proxy_http_port|default('8080') }}
+          protocol: TCP
+        - containerPort: 443
+          hostPort: {{ xsnippet_proxy_https_port|default('4443') }}
```

Comment:
nit: Is `4443` a common pattern? I thought `8443` is more common.

Reply:
No clue :) I've never used either. `8443` sounds good to me.